include/libtorrent/kademlia/get_item.hpp

```diff
@@ -43,7 +43,7 @@ namespace libtorrent { namespace dht
 class get_item : public find_data
 {
 public:
-	typedef boost::function<bool(item&)> data_callback;
+	typedef boost::function<bool(item&, bool)> data_callback;
```

Comment:
sorry. what do you think about maybe making this bool an int instead, and pass in flags? It would make it a lot easier to extend with more information, and it would be ABI compatible.